core/src/main/java/org/mule/runtime/core/internal/context/DefaultMuleContext.java

```diff
@@ -266,8 +266,8 @@
 
   static {
     // Log dropped events/errors
-    Hooks.onErrorDropped(error -> LOGGER.debug("ERROR DROPPED " + error));
-    Hooks.onNextDropped(event -> LOGGER.debug("EVENT DROPPED " + event));
+    Hooks.onErrorDropped(error -> LOGGER.debug("ERROR DROPPED {}", (Object) error));
```

Comment:
i would log the exception with the full stack trace here. It is already in DEBUG level and the information provided by it may be useful.

Reply:
Agree (I have been overly conservative).